src/utils/SlotComponent.js

```diff
@@ -1,4 +1,9 @@
-import { h as createElement } from 'vue'
+import { h as createElement, onUpdated } from 'vue'
+import {
+    camelize,
+    hyphenate,
+    toHandlerKey
+} from '@vue/shared' // eslint-disable-line vue/prefer-import-from-vue
```

Comment:
Do we still need to have the disable ES-lint comment here?

Reply:
@wesdevpro 
> Do we still need to have the disable ES-lint comment here?

Without that comment, we get the following error:
>   6:8  error  Import from 'vue' instead of '@vue/shared'  vue/prefer-import-from-vue

Unfortunately, they are not exported from `vue`, so I had to import them from `@vue/shared`.